### DISCUSSION TITLE
OpenAssistant / oasst-sft-4-pythia-12b-epoch-3.5  support

### DISCUSSION THREAD

**Discussion by jurandfantom** [oobabooga/text-generation-webui#1251] [Ideas]
No clue how github works so hopefully I doing that in correct way. 
New interesting project was released and would be great to use it with oobabooga

1) YT: https://www.youtube.com/watch?v=ddG2fM9i4Kk
2) HF: https://huggingface.co/OpenAssistant/oasst-sft-4-pythia-12b-epoch-3.5

If its possible to use it right away, let me know  (how) :)

**Comment by BarfingLemurs** [comment]
any models already supported should work.

**Comment by ExtinctionEvent** [comment]
It requires a large amount of VRAM. So load in 8bit. Load with...
python server.py --model oasst-sft-4-pythia-12b-epoch-3.5   --load-in-8bit
